lib/modules/versioning/pvp/readme.md

```diff
@@ -0,0 +1,15 @@
+[Package Versioning Policy](https://pvp.haskell.org/) is used with Haskell.
+It's like semver, except that the first _two_ components are parts of the major
+version. That is, in `A.B.C`:
+
+- `A.B`: major version
+- `C`: minor
+
+The remaining components are all considered parts of the patch version, and
+they will be concatenated to form a `number`, i.e. IEEE 754 double. This means
+that both `0.0.0.0.1` and `0.0.0.0.10` have patch version `0.1`.
+
+The range syntax comes from Cabal, specifically the [build-depends
+section](https://cabal.readthedocs.io/en/3.10/cabal-package.html).
+
+This module is considered experimental since it has limited range support.
```

Comment:
Document what ranges are supported.

Reply:
Added in [ffea9b2](https://github.com/renovatebot/renovate/pull/32298/commits/ffea9b2a8db7d701c5074de5dceb8b65813aeb56), thanks for the suggestion